QUICKSTART.md

```diff
@@ -122,16 +122,16 @@ void b(Span parentSpan) {
 ```
 The OpenTelemetry API offers also an automated way to propagate the parent span on the current thread:
 ```java
-void a() {
-  Span parentSpan = tracer.spanBuilder("a").startSpan();
+void c() {
+  Span parentSpan = tracer.spanBuilder("c").startSpan();
   try(Scope scope = parentSpan.makeCurrent()) {
-    b();
+    d();
   } finally {
     parentSpan.end();
   }
 }
-void b() {
-  Span childSpan = tracer.spanBuilder("b")
+void d() {
+  Span childSpan = tracer.spanBuilder("d")
```

Comment:
Please avoid 1 character function names in examples, also using `b` and `d` makes things even harder to see.

Reply:
I thought you liked c!  ;) 

I'll clean this up.

Reply:
By the way, the only reason this even got touched is that I'm starting to think about auto-generating these docs from a template java file, and I needed it all to compile locally. :)

Reply:
done